src/codewhisperer/util/telemetryHelper.ts

```diff
@@ -299,6 +299,7 @@ export class TelemetryHelper {
             codewhispererSupplementalContextTimeout: supplementalContextMetadata?.isProcessTimeout,
             codewhispererSupplementalContextIsUtg: supplementalContextMetadata?.isUtg,
             codewhispererSupplementalContextLength: supplementalContextMetadata?.contentsLength,
+            codewhispererSupplementalContextStrategyId: supplementalContextMetadata?.strategy,
```

Comment:
Since this name is from the telemetry definitions we will have to ignore the lint rule in this case.
```suggestion
            // eslint-disable-next-line @typescript-eslint/naming-convention
            codewhispererSupplementalContextStrategyId: supplementalContextMetadata?.strategy,
```

In the future, such long names should not be used. There really is no reason that names need to be that long.

Reply:
yea https://github.com/aws/aws-toolkit-common/commit/a826622c0d6544cb2da0abe87f52e7105cb2f971#diff-f9ad596fbedec6dc08aa26f65b8a0f9358ae2be13476b00a82557f7d7125c568R609

Reply:
oops should be `eslint-disable-next-line id-length`. i pushed a fix